subsys/mgmt/mcumgr/lib/smp/src/smp.c

```diff
@@ -53,22 +63,20 @@ smp_rsp_op(uint8_t req_op)
 	}
 }
 
-static void
-smp_make_rsp_hdr(const struct mgmt_hdr *req_hdr, struct mgmt_hdr *rsp_hdr, size_t len)
+static void smp_make_rsp_hdr(const struct smp_hdr *req_hdr, struct smp_hdr *rsp_hdr, size_t len)
 {
-	*rsp_hdr = (struct mgmt_hdr) {
+	*rsp_hdr = (struct smp_hdr) {
 		.nh_len = len,
 		.nh_flags = 0,
 		.nh_op = smp_rsp_op(req_hdr->nh_op),
```

Comment:
In this case, does it make more sense to get rid of `smp_hdr_hton()` entirely and use the `sys_*()` functions instead?

Reply:
Yes.